config/locales/en.yml

```diff
@@ -34,7 +34,6 @@ en:
             invalid_requirements_types:
               one: 'requirements.json contains an invalid type: %{invalid_types}'
               other: 'requirements.json contains invalid types: %{invalid_types}'
-            multiple_channel_integrations: 'Specifying multiple channel integrations in requirements.json is not supported.'
```

Comment:
That darn @timbertson, can't read:
`# This file is generated by rake i18n:standardize. Do NOT edit it directly.`
;-)